Lib/shutil.py

```diff
@@ -1458,7 +1458,7 @@ def _win_path_needs_curdir(cmd, mode):
     if we should add the cwd to PATH when searching for executables if
     the mode is executable.
     """
-    return (not (mode & os.X_OK) or
+    return (not (mode & os.X_OK)) or \
```

Comment:
PEP 8 discourages the use of backslash line continuations:

>The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

Format the statement however you like that agrees with PEP 8 (i.e. prefer formatting that's encouraged, and avoid formatting that's discouraged). But ensure that it short circuits when `mode` doesn't include `os.X_OK`.